### ISSUE TITLE
RIP Anything.js

### ISSUE THREAD

**Issue by gytdau** [Rabrennie/anything.js#312]
![Status](http://i.imgur.com/feS28Ys.png)

Goodbye, sweet love. You lived a short but eventful life. Amen.

**Comment by thehowl** [comment]
Sounds like a great time to post on Hacker News!

**Comment by thehowl** [comment]
(and then make it a startup)

**Comment by vccortez** [comment]
Never gonna give you up, never gonna let you down. Guys, pls, don't let this die, it's my main source of commits. :sob: pls lemme smash my keyboard.

**Comment by jslegers** [comment]
Can anyone explain what's the idea behind this project?!

**Comment by gytdau** [comment]
Did you read README.md?

On Monday, 14 December 2015, John Slegers notifications@github.com wrote:

> Can anyone explain what's the idea behind this project?!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Rabrennie/anything.js/issues/312#issuecomment-164375688
> .

## 

Thanks,
_Gytis Daujotas_

**Comment by jslegers** [comment]
@gytdau :

I did... and it didn't make any sense to me ;-)

**Comment by gytdau** [comment]
The idea is that we accept any pull request into the repository, as long as
it follows the short list of rules.

On Monday, 14 December 2015, John Slegers notifications@github.com wrote:

> @gytdau https://github.com/gytdau :
> 
> I did... and it didn't make any sense to me ;-)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Rabrennie/anything.js/issues/312#issuecomment-164390102
> .

## 

Thanks,
_Gytis Daujotas_

**Comment by jslegers** [comment]
@gytdau :

I got that. I just don't see the point of collaborating on such a virtual Frankenstein.
Should I see this a social experiment?

**Comment by gytdau** [comment]
Kind of. It's more of a "do it just because you can" thing.

On Monday, 14 December 2015, John Slegers notifications@github.com wrote:

> @gytdau https://github.com/gytdau :
> 
> I got that. I just don't see the point of collaborating on such a virtual
> Frankenstein.
> Should I see this a social experiment?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Rabrennie/anything.js/issues/312#issuecomment-164396172
> .

## 

Thanks,
_Gytis Daujotas_

**Comment by jslegers** [comment]
Hmmm...
Maybe I'm just getting too old for this stuff :-p

![Yes, we can!](http://i65.tinypic.com/wbx8id.jpg)

**Comment by Rabrennie** [comment]
RIP In Pepperoni

**Comment by gytdau** [comment]
Rest in peace in pepperoni with pepper.

RIPIPWP.

On Monday, 14 December 2015, Rab Rennie notifications@github.com wrote:

> RIP In Pepperoni
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Rabrennie/anything.js/issues/312#issuecomment-164431732
> .

## 

Thanks,
_Gytis Daujotas_

**Comment by seiyria** [comment]
No just pepperoni is fine, you savages.

**Comment by gytdau** [comment]
No pepporoni left, sir. @Rabrennie ate all of it again. Would you settle for three sliced rats instead?

**Comment by seiyria** [comment]
Man, I guess I have to. I won't like it, though!

**Comment by Rabrennie** [comment]
I thought my inbox had been freed from the github spam. I guess not :(

**Comment by gytdau** [comment]
You will never be free. Anything.js will follow you until the end of the earth. Ding. That's an alert for a new pull request. And you thought the repo was dead. __Ding__. It doesn't stop. You will be bound eternally to Anything.js. _Ding_. It's t̶re̴n͠di̡n҉g ͠on ̧Ŗe̢ddi͢t a͟gain. __DING__. Yo̶ur̛ ̡inbox̴ nev̸eŗ be͝ r̛ȩleas͝e͟d f̴rom̶ Luci̸fe̶r's͘ g̡rasp.͡ **DING** Untìl the ͠end̨ of ti̛m̧e̛,͠ yo͜u ͟śha̕l̶l sùf͝fe̕r ͢the be͘ļl͢s.̀ T͏h͏e rin͝gi͘ng͝ ̸wi͞l͞l͘ not s̶t͞op͠ f͡o͞r ̷àn̛y ͡mo͝rt͏a͠l. **DING** YO͘U̡ WI͘LL ̛ŚUF̕FER ̨FOR̀ ̨W̢H҉AT҉ ͘Y͠O͞U HAVE̵ U͟ǸL͡EA̶SH͠ED͢.̵ **DING** TH̵E ̸ĆH̸͘I͏M̧̀É͡S͏ ̢̧̀F̀OL͜L̵̨̛OW̸ ̵̷Y̛̛O͝Ų̕͢.̵**DING**TH̸̢Ę̴ ́͘͡N̸̢Ǫ̸TÌ҉F҉̵I̴̢C͜À͠T̷͏҉Í̛O̶͟N̸̛S̶͡ ̛͏̷NȨ̀V͜E̸̛͝R҉͜͢ ͘Ę̕͟N̕D̵͡ **DING DING DING DING**D̵̴͢O҉M͝I͠N̶̕IU̧Ś̢͞ ̧҉DI̸̧A̢̨̧B͞Ò̷́L̷̀͟U̷̕S͠ G͟I̷̕͢T̡H̡͞UB̶̀U̷S͡.̢̛͞

_ding_

**Comment by TheBITLINK** [comment]
RIP in RIP

**Comment by vccortez** [comment]
Hey guys, I'm back. What did I miss?  
![Oh Damn](http://i.imgur.com/cMWalaa.gif)

**Comment by Rabrennie** [comment]
@vekat pretty much

**Comment by TheBITLINK** [comment]
As i said before...

Gentlemen, it was nice to code with you...

**Comment by gytdau** [comment]
Welcome back to Anything Report, folks.

As you can see on ![this page](https://github.com/Rabrennie/anything.js/graphs/contributors), the contribution graphs are forming a nice, pleasing bell shape, as most contributors are slowing down. When you scroll down, you can see many of the contributors that submitted a joke because they saw it on Reddit, and afterwards, got bored of it almost immediately. The graphs of those contributors are forming thin lines instead. The lowest ranking medal goes to #93rd place - @sxqwaade - with his, oddly enough, 165 lines of contributions.

Now, onto the ![traffic page](https://github.com/Rabrennie/anything.js/graphs/traffic) - you can see that every graph has already approached zero. Viewcounts and clones have fallen to a negligible amount. This spells out _bad news_ for the Anything.js repository, as less contributions = less views = less contributions, on and on in a vicious cycle.

That's all the reporting for today, folks.

Stay tuned for Anything.js's official funeral and cremation!

Over and out.

**Comment by vccortez** [comment]
I would still be submitting a joke a day if I wasn't having internet issues. :sob:

**Comment by Rabrennie** [comment]
@vekat don't worry about it :+1:

**Comment by MrOnosa** [comment]
I added Open Source Contributor to my linked in profile thanks to Anything.js. If it goes under, I'll have nothing to back up this claim. Come on people! Let's get to work!

**Comment by gytdau** [comment]
Did someone called Tyler just write a message here and almost immediately delete it? He said:

```
I added Open Source Contributor to my linked in profile thanks to Anything.js. If it goes under, I'll have nothing to back up this claim. Come on people! Let's get to work!
```

COME BACK TYLER
WE LOVE YOU

**Comment by seiyria** [comment]
He definitely did. I was going to say to Do Anything (tm)!

**Comment by gytdau** [comment]
I'm starting an investigation to find Tyler. He said he's a contributor, so it should be easy.

**Comment by gytdau** [comment]
Nope, after checking every contributor profile, there's no Tyler.

God damn, Typer. You slipped out of my hands this time. But one day I will catch you Tyker... one day...

**Comment by MrOnosa** [comment]
Tyler here. It was a lie, people. I did contribute to anything.js when it was new, but I didn't put it on my linked in. 

_shame_  . . _shame_  . . . . .  _shame_
  . .   _shame_     . . . . .     _shame_
  _shame_     . . . . .    . . . . .   _shame_
         . . . . .    . . . . .          _shame_    _shame_

**Comment by seiyria** [comment]
we have you now, tyler

one of us

**Comment by gytdau** [comment]
Tyler... oh Tyler.

My sweet summer child. You silly boy.

You fell straight into our trap.

Now you will be banished to contribute to the project forever.

**Comment by MrOnosa** [comment]
Ha! So, I wrote the original version of  https://github.com/Rabrennie/anything.js/blob/master/src/async_doThingTomorrow.js which was converted by @gytdau. Thanks for the credit. Jerk. (I'm kidding, you're my new bff)

**Comment by gytdau** [comment]
Question. Is your last name Onosa?

**Comment by MrOnosa** [comment]
Nope. Back in the day, I played Final Fantasy 11. My online handle was randomly generated as Onosa. So, I used Onosa as my online handle wherever I went. Eventually, Onosa was taken somewhere, so I used MrOnosa instead. I liked it better than Onosa1 or whatever. But, it is confusing.

**Comment by jslegers** [comment]
![Yes, we can!](http://www.trbimg.com/img-5762ff99/turbine/la-1466105938-snap-photo/650/650x366)

![Yes, we can!](http://i.imgur.com/6hfXlB3.png)

![Yes, we can!](http://images.popmatters.com/news_art/t/the-rough-guide-to-psychedelic-india.jpg)

![Yes, we can!](http://orig03.deviantart.net/e2b0/f/2013/192/3/d/cats_and_cats_and_cats_by_liquid_mushroom-d6d0zty.jpg)

![Yes, we can!](http://cdn.quotesgram.com/img/19/3/1344654032-15857-tyler-durden-quote-1920x1080-typography-wallpaper.jpg)

**Comment by gytdau** [comment]
okay then

**Comment by thehowl** [comment]
k

On 30 June 2016 18:57:10 CEST, Gytis Daujotas notifications@github.com wrote:

> okay then
> 
> ---
> 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/Rabrennie/anything.js/issues/312#issuecomment-229720879

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by SCKelemen** [comment]
;.(

**Comment by Rabrennie** [comment]
Pls contribute

**Comment by gytdau** [comment]
Please. Or else Rab's mother will consider him a failure. You don't want
that. Do you?

On Thursday, 30 June 2016, Rab Rennie notifications@github.com wrote:

> Pls contribute
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/Rabrennie/anything.js/issues/312#issuecomment-229771424,
> or mute the thread
> https://github.com/notifications/unsubscribe/AIaiA-mKb8t6Qnex_LEUB9g10oEJYL4Yks5qRCBygaJpZM4G0RZD
> .

## 

Thanks,
_Gytis Daujotas_

**Comment by seiyria** [comment]
you don't know what I want

**Comment by vccortez** [comment]
Sup guys?

**Comment by TheBITLINK** [comment]
OOOOOOO
Did this project just came back from the depths of hell?

**Comment by seiyria** [comment]
no

**Comment by TheBITLINK** [comment]
![145757337719](https://cloud.githubusercontent.com/assets/3434300/16509187/4e541500-3f08-11e6-91e4-4ed89307a390.jpg)

**Comment by vccortez** [comment]
Welcome back to life!

**Comment by Rabrennie** [comment]
# **_Think we need a CSS theme to go along with anything.js_**

![giphy](https://cloud.githubusercontent.com/assets/9087372/16542507/08f0aa22-40a3-11e6-83b9-dcd3518195fd.gif)

**Comment by TheBITLINK** [comment]
# anything.css confirmed?

![cy](https://cloud.githubusercontent.com/assets/3434300/16542563/9603cb24-407b-11e6-8486-9e3204f3d7b1.png)

**Comment by Rabrennie** [comment]
@TheBITLINK  https://github.com/Rabrennie/anything.css/blob/master/README.md

**Comment by TheBITLINK** [comment]
Y E S

**Comment by vccortez** [comment]
Sorry, I only do my CSS _inside_ my JavaScript.

**Comment by Rabrennie** [comment]
![giphy 1](https://cloud.githubusercontent.com/assets/9087372/16542821/2d7c76f6-40b1-11e6-8494-a42304ce1548.gif)

**Comment by jslegers** [comment]
![Yes, we can!](http://www.flipflop-gypsies.com/wp-content/uploads/2015/06/yikes.jpg)

**Comment by SCKelemen** [comment]
This is most definitely still alive. Bringing nightmares to child-men everywhere.

**Comment by vccortez** [comment]
Wuuuuut.

**Comment by TheBITLINK** [comment]
hi how is it going

**Comment by seiyria** [comment]
It's been a while, how's it been over there?

**Comment by SCKelemen** [comment]
https://www.youtube.com/watch?v=Y6ljFaKRTrI

**Comment by Rabrennie** [comment]
😢

**Comment by gytdau** [comment]
I know anything.js was like a child to you. I can only imagine the pain.

**Comment by Rabrennie** [comment]
😭

**Comment by JCharante** [comment]
@vccortez 
> Sorry, I only do my CSS _inside_ my JavaScript.

laughs in react tailwind

Oh boy, we were so innocent back then
